avocado/plugins/jobscripts.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core.settings import settings
+from avocado.plugins.base import JobPre, JobPost
+
+
+CONFIG_SECTION = 'avocado.plugins.jobscripts'
```

Comment:
Definitely. I used the `plugins.` prefix for other work, this is indeed inconsistent.